x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -115,12 +115,10 @@ public EqlSearchRequest(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_7_15_0)) {
             this.ccsMinimizeRoundtrips = in.readBoolean();
         }
-        if (in.getVersion().onOrAfter(Version.V_8_0_0)) { // TODO: Remove after backport
```

Comment:
I guess the check could be removed, but I'd find it more correct if we specified the version when these params were introduced (`V_7_9_0`?).
And similar for writing the values.
But this is more efficient like this too, given that comms with a version before 7.9.0 wouldn't [need to] work anyways.

Reply:
Exactly, since we do not support older versions, all these checks should eventually be removed probably, it would make the code more clear (in terms of efficiency, I'm pretty sure the difference is minimal)